spectroscopy/spectra_generator.md

```diff
@@ -265,12 +272,8 @@ We show flux in mJy as a function of time for all available bands for each objec
 ```python
 ### Plotting ####
 create_figures(df_spec = df_spec,
-             bin_factor=10,
+             bin_factor=5,
```

Comment:
something is wrong with the X-axis wavelengths of the JWST spectra.  I am getting the clear prism from 0.0001 to 0.0005 microns. I'm guessing there is a unit problem here.  

Also, can the plotting be changed to eliminate the downward excursions on the log plots? ie, are there zeros in the spectra or are those error bars that extend to the bottoms of the log plots? or???

Reply:
1. I changed the plotting. Opened a can of worms, the units were somehow wrong. I made it consistent now by using the astropy units throughout.
2. these are very small values that are making the plot ugly. I can try to sigma-clip them out...

Reply:
for #2, I sigma clipped and also removed negative fluxes (just for plotting)